docs/team/sueann-chua.md

```diff
@@ -15,6 +15,14 @@ Given below are my contributions to the project.
 [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=sueann-chua&tabRepo=AY2122S1-CS2103T-W13-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
 
 ### 2. Enhancements implemented
+
+As mentioned below, I helped implement the features for `edit_p` and `edit_i` and helped in bugfixes for`edit_c`, 
+`edit_p` and `edit_i` as well. The edit features were sometimes a challenge to implement due to the cascading effects
+on the other objects when one object was edited. For example, editing a position would mean that the candidates
+applied to the position, and the interviews scheduled for that position would need to be updated as well. There were 
+some instances in which I had missed out certain possibilities and this resulted in certain bugs that were eventually 
+fixed.
```

Comment:
There are some extra line breaks between line 18 and 24. Were they caused by or auto wrapping?

Reply:
Line breaks doesn't affect the final renderindg without a <br> tag. This chunk of words should be broken up to multiple points though

Reply:
split the chunk into two for readability. extra line breaks were just to help me read it more easily without scrolling but does not affect the actual line breaks.